staker-rewards/lib/requirements.js

```diff
@@ -135,9 +161,34 @@ export default class Requirements {
     this.context.cache.storeTransactions(transactions)
   }
 
-  async checkAuthorizations(operator) {
-    console.debug(`Checking authorizations for operator ${operator}`)
+  async check(operator) {
+    console.log(`Checking requirements for operator ${operator}`)
+
+    const {
+      factoryAuthorizedAtStart,
+      poolAuthorizedAtStart,
+      poolDeauthorizedInInterval,
+    } = await this.checkAuthorizations(operator)
+
+    const minimumStakeAtStart = await this.checkMinimumStakeAtIntervalStart(
+      operator
+    )
+
+    const minimumUnbondedValueRegisteredAtStart = await this.checkUnbondedValueRegisteredAtIntervalStart(
```

Comment:
Thoughts on naming it `inPoolIfRequired`? What we are checking here actually is whether the operator is in the sortition pool if it should be there:
> If the operator has more ETH_unbonded than the minimum unbonded value and is not in the sortition pool at T_start, they are not getting any rewards for interval T.

Reply:
Yeah, I didn't like the current name either.